lib/tasks/migrate.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+namespace :migrate do
+  desc 'Migrate legacy file attachments into ActiveStorage'
+  # Usage: rake migrate:dissertation_files
+  # Migration of dissertation files from legacy attachments to ActiveStorage
+  # This can safely be re-run multiple times
+  task dissertation_files: :environment do
+    LegacyDissertationFile.find_each do |dissertation_file|
+      next if dissertation_file.submissions.blank?
+      next if dissertation_file.submissions.first.dissertation_file.attached?
```

Comment:
Worth the effort to add some logging in these cases?

Reply:
I wouldn't think so, as this should indicate a new submission or an already migrated submission, but I could be swayed to log "not migrating..." or something to that effect.

Reply:
Updated selection process.